app/src/main/java/chat/rocket/android/chatroom/adapter/ChatRoomAdapter.kt

```diff
@@ -214,7 +214,13 @@ class ChatRoomAdapter(
             val temp = action as ButtonAction
             if (temp.url != null && temp.isWebView != null) {
                 if (temp.isWebView == true) {
-                    //TODO: Open in a configurable sizable webview
+                    //Open in a configurable sizable webview
+                    when(temp.webViewHeightRatio){
+                        "full" -> openFullWebPage(temp, roomId)
```

Comment:
What about creating a const with this value and using it in every place on you code?

Reply:
@filipedelimabrito I think this property value is not known at compile time to make it `const`. Correct me If I am wrong.

Reply:
@Cool-fire It has nothing about compile time. Make it a const since it is a const.

Reply:
@filipedelimabrito I am getting not why to make `temp.webViewHeightRatio` const. The webviewHeightRation has values tall, full, compact. How can it be a const?. Please correct me if I am wrong.

Reply:
https://github.com/RocketChat/Rocket.Chat.Android/pull/2327/commits/a45ead7718ea1af959b4a55e7677e8a73d95c7f7